pkgs/applications/editors/emacs/macport-25.1.nix

```diff
@@ -48,7 +48,7 @@ stdenv.mkDerivation rec {
     "--enable-mac-app=$$out/Applications"
   ];
 
-  CFLAGS = "-O3 -DMAC_OS_X_VERSION_MAX_ALLOWED=1090";
+  CFLAGS = "-O3 -DMAC_OS_X_VERSION_MAX_ALLOWED=1090 -DMAC_OS_X_VERSION_MIN_REQUIRED=1090";
```

Comment:
I would say to just merge this now. If you can merge #20536 after this, it should fix emacs24Macport as well